follower/consensus_follower.go

```diff
@@ -191,25 +195,21 @@ func (cf *ConsensusFollowerImpl) AddOnBlockFinalizedConsumer(consumer pubsub.OnB
 
 // Run starts the consensus follower.
 func (cf *ConsensusFollowerImpl) Run(ctx context.Context) {
-	runAccessNode(ctx, cf.NodeBuilder)
-}
-
-func runAccessNode(ctx context.Context, anb *access.UnstakedAccessNodeBuilder) {
 	select {
 	case <-ctx.Done():
 		return
 	default:
 	}
 
 	select {
-	case <-anb.Ready():
-		anb.Logger.Info().Msg("Access node startup complete")
+	case <-cf.Ready():
+		cf.Logger.Info().Msg("Access node startup complete")
 	case <-ctx.Done():
-		anb.Logger.Info().Msg("Access node startup aborted")
+		cf.Logger.Info().Msg("Access node startup aborted")
```

Comment:
this is no longer to do with `Access node`, better to update the comment

Reply:
addressed in 0243920